.github/workflows/label-analysis.yml

```diff
@@ -28,30 +28,3 @@ jobs:
           done
           echo "PR ${{github.event.number}} requires at least one non-triage team assignment label (label starting by 'team/')"
           exit 1
-  skip-qa:
-    if: github.triggering_actor != 'dd-devflow[bot]'
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check ability to skip qa
-        run: |
-          skip=0
-          done=0
-          noChange=0
-          labels="${{join(github.event.pull_request.labels.*.name, ' ')}}"
-          for label in $labels; do
-            if [[ "$label" == "qa/skip-qa" ]]; then
-              skip=1
-            fi
-            if [[ "$label" == "qa/done" ]]; then
-              done=1
-            fi
-            if [[ "$label" == "qa/no-code-change" ]]; then
-              noChange=1
-            fi
-          done
-          if [[ ($skip -eq 0 && $done -eq 0 && $noChange -eq 0) || ($skip -eq 1 && $done -ne $noChange) ]]; then
-            exit 0
-          else
-            echo "Skip QA labels are not correct on PR ${{github.event.number}}. You must set 'qa/skip-qa' with exacly 1 justification, 'qa/done' or 'qa/no-code-change'"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Can you keep this check to ensure `qa/[done|no-code-change]` labels are not set together ? Meaning they do not appear on the same PR at the same time

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Does this really change something ? Either way the code doesn't need QA card ?
Asking this because we're supposed to be deprecate those labels soon

Reply:
It does not change the usage of those labels, it changes their tracing - when observing PRs, sometime they come up with both labels, and we want to track how many PRs that would have required QA (so no `qa/no-code-change`) did shift the QA to the left (so have `qa/done`). If both are there it is confusing

Reply:
Makes sense, changing this then

Reply:
This check is currently broken, it's not possible to restart it cleanly except by closing the PR.. Do you think we could remove it for now, and then implement what is required for the tracking in the future ? It would result in a clear win for everyone

Reply:
> This check is currently broken
@paulcacheux WDYM ? Are you talking about #22646 ?

Reply:
Yes, you can now disregard :)

Reply:
Thanks !